internal/cluster/validations/validation_test.go

```diff
@@ -679,35 +681,35 @@ var _ = Describe("IPv6 support", func() {
 	}
 })
 
-var _ = Describe("Network amount and order", func() {
+var _ = Describe("Machine Network amount and order", func() {
 	tests := []struct {
-		element []*string
+		element []*models.MachineNetwork
 		valid   bool
 	}{
 		{
-			element: []*string{swag.String("1.2.5.0/24"), swag.String("1002:db8::/119")},
+			element: []*models.MachineNetwork{{Cidr: "1.2.5.0/24"}, {Cidr: "1002:db8::/119"}},
 			valid:   true,
 		},
 		{
-			element: []*string{swag.String("1002:db8::/119"), swag.String("1.2.5.0/24")},
+			element: []*models.MachineNetwork{{Cidr: "1002:db8::/119"}, {Cidr: "1.2.5.0/24"}},
 			valid:   false,
 		},
 		{
-			element: []*string{swag.String("1.2.5.0/24"), swag.String("1002:db8::/119"), swag.String("1.2.6.0/24"), swag.String("1.2.7.0/24")},
-			valid:   true,
+			element: []*models.MachineNetwork{{Cidr: "1.2.5.0/24"}, {Cidr: "1002:db8::/119"}, {Cidr: "1.2.6.0/24"}, {Cidr: "1.2.7.0/24"}},
+			valid:   false,
 		},
 		{
-			element: []*string{swag.String("1002:db8::/119"), swag.String("1.2.5.0/24"), swag.String("1.2.6.0/24"), swag.String("1.2.7.0/24")},
+			element: []*models.MachineNetwork{{Cidr: "1002:db8::/119"}, {Cidr: "1.2.5.0/24"}, {Cidr: "1.2.6.0/24"}, {Cidr: "1.2.7.0/24"}},
 			valid:   false,
 		},
 	}
 	for _, t := range tests {
 		t := t
```

Comment:
```suggestion
	for _, test := range tests {
		t := test
```

I'm surprised with all the linting checks for shadowing that this is allowed. A little nitpicky but I think we should avoid this if we can.

Reply:
As previously, I see this pattern in 5 different tests in this file. Do we want to change it in here or follow the current style ? (cc @flaper87)

Reply:
I guess we can change this here but definitely let's not change all the cases like this in the file. Let's keep the backport as small as possible.

Reply:
Fixed